blivet/devicefactory.py

```diff
@@ -1168,7 +1173,8 @@ def configure(self):
                 self.storage.format_device(member, get_format("luks",
                                                               min_luks_entropy=self.min_luks_entropy,
                                                               luks_version=self.luks_version,
-                                                              pbkdf_args=self.pbkdf_args))
+                                                              pbkdf_args=self.pbkdf_args,
+                                                              sector_size=self.luks_sector_size))
```

Comment:
You're passing `sector_size`  but the format class expects `luks_sector_size`.

Reply:
(Same throughout this file)

Reply:
Oops. Good catch. Fixed (and I also added the sector size change to devicefactory test case).